routes/persoon.ts

```diff
@@ -0,0 +1,30 @@
+import Router from 'express-promise-router';
+
+import { Request, Response } from 'express';
+
+import { STATUS_CODE } from '../util/constants';
+import { persoonUsecase } from '../controllers/persoon';
+
+export const personenRouter = Router();
+
+personenRouter.get(
```

Comment:
Endpoint is not currently used in the frontend but this will be when adding the logic for the dynamic fractie options. 

The getFractie method on person is used in another controller so this is just prep  + handy to verify current logic with postman ;p